tokens/contracts/access/MinterAccessControl.sol

```diff
@@ -0,0 +1,51 @@
+// SPDX-License-Identifier: MIT
+
+pragma solidity >=0.6.0 <0.8.0;
+
+import "@openzeppelin/contracts-upgradeable/proxy/Initializable.sol";
+import "@openzeppelin/contracts-upgradeable/access/OwnableUpgradeable.sol";
+import "@openzeppelin/contracts-upgradeable/utils/EnumerableSetUpgradeable.sol";
+
+abstract contract MinterAccessControl is Initializable, OwnableUpgradeable {
+    using EnumerableSetUpgradeable for EnumerableSetUpgradeable.AddressSet;
+
+    EnumerableSetUpgradeable.AddressSet private _minters;
+    
+    event MinterGranted(address indexed account);
```

Comment:
1.I think is better to change "MinterGranted" for  MinterAdded
2.Lets think togrther, do we need to emit `operator` (_msgSender()), in event parameters like that:
event MinterAdded(address indexed operator, address indexed newMinter ) 
3.need test for cheking emit event

Reply:
3 points done